  public void testBase() throws Exception {
    doTest("base");
  }

